package.json

```diff
@@ -27,8 +27,18 @@
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/preset-env": "^7.8.3",
     "@babel/preset-react": "^7.8.3",
+    "@storybook/addon-actions": "^5.3.6",
+    "@storybook/addon-docs": "^5.3.6",
+    "@storybook/addon-knobs": "^5.3.6",
+    "@storybook/addon-links": "^5.3.6",
+    "@storybook/addon-notes": "^5.3.6",
```

Comment:
Didn't see you initialize this addon in the config. Are you still using it or is it a peer dependency for one of the others?

Reply:
Yep, good find. I've nixed them